fetch/api/request/request-error.html

```diff
@@ -27,8 +27,8 @@
       },"Input URL has credentials");
 
       test(function() {
-        assert_throws(new TypeError() , function() { new Request("", {"mode" : "navigate"}); },
-          "Expect TypeError exception");
+        var req = new Request("", {"mode" : "navigate"});
+        assert_equals(req.mode, "same-origin", "navigate mode in initializer should become same-origin");
```

Comment:
I would remove that test and update fetch/api/request/request-init-001.sub.html instead (check for "var modes =")